packages/ui/src/components/Button/Button.stories.tsx

```diff
@@ -2,55 +2,51 @@ import React from 'react';
 import { Meta, Story } from '@storybook/react';
 import { Button, ButtonProps } from '.';
 import { Facebook } from '@styled-icons/boxicons-logos/Facebook';
-
-const Template: Story<ButtonProps> = (args) => (
-  <Button {...args}>Call to action </Button>
-);
+import styled from 'styled-components';
+
+type Sizes = ['xs', 'sm', 'md', 'lg'];
+const sizes: Sizes = ['xs', 'sm', 'md', 'lg'];
+
+const ButtonsSeparator = styled.div`
+  margin-bottom: 15px;
```

Comment:
I would encourage you to start using derivatives of 4px. ;)

Reply:
Shit for the margins too!! My baad, So padding, margins, what else? :D

Reply:
Not so important ;)